### DISCUSSION TITLE
Embedding studio.foxglove.dev

### DISCUSSION THREAD

**Discussion by ninoweg** [foxglove/community#937] [General]
It doesn't appear to be allowed to embed studio.foxglove.dev on another website. At least, I wasn't able to embed it within an <iframe>. Is there another domain or another possibility to embed Foxglove?

**Comment by amacneil** [comment]
It's not currently supported on our official hosted version due to security issues (clickjacking, etc). We would like to support that in the future but do not have any current plans.

Depending on what you want to do, your other option is to host your own copy of the open source version, e.g. using our [docker image](https://github.com/foxglove/studio/pkgs/container/studio).

If you're looking to do this in a commercial setting please feel free to [contact us](https://foxglove.dev/demo) and we're happy to talk through options.

**Reply by ninoweg** [reply]
Self hosting using docker is just what I was looking for. Thank you!
